lib/measured/measurable.rb

```diff
@@ -1,6 +1,26 @@
 class Measured::Measurable < Numeric
   include Measured::Arithmetic
 
+  PARSE_REGEX = /
+    \A         # beginning of input
+    \s*        # optionally any whitespace
+    (          # capture the value
+      -?       # number can be negative
+      \d+      # must have some digits
+      (?:      # do not capture
+        \.     # period to split fractional part
```

Comment:
We should permit the use of commas in place of periods. (https://github.com/Shopify/measured/pull/81#discussion_r102552174)